webserver/locust-metrics/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- ../../voting-webapp/application
+- ./../voting-webapp/application
```

Comment:
I am noticing that some places include relative paths, and others use fully qualified paths on GitHub.

Is there a convention we can settle on here for future? I usually only ever use fully qualified GitHub paths if I am using a kustomization from another repo. Here, we are sometimes referencing thestormforge/examples repo with the fully qualified method, which doesn't make sense for our users who are going to clone the whole repo anyway.

Reply:
If someone were to `kustomize build github.com/thestormforge/examples/webserver/locust-metrics` it would effectively clone `examples` to a temp directory (I believe `github.com/thestormforge/examples//webserver/locust-metrics` is or was different in that it would only keep the subdirectory; the behavior is specific based on the version of Kustomize being used and the exact syntax of the reference—for a while there were conflicting Git implementations between what Kustomize had implemented itself and what was in the go-getter fork they used). If this reference were a Git reference, I'm pretty sure it would result in a second temporary clone being made.

Originally, I believe the Git references were used in places where the loading restrictions prevented Kustomize from building successfully using the default settings; they were also used in the READMEs so people wouldn't need to manually clone the repository.

Reply:
Also, isn't `./..` the same as `../` ?